Source: David Legrand <david.legrand@clever-cloud.com>
Upstream: Yes (#21094)
Reason: Prevent modules updates during `v build-tools`, which could require network access.
---
 cmd/tools/vbuild-tools.v | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/cmd/tools/vbuild-tools.v b/cmd/tools/vbuild-tools.v
index a417cd3c1..a88ccb823 100644
--- a/cmd/tools/vbuild-tools.v
+++ b/cmd/tools/vbuild-tools.v
@@ -19,7 +19,9 @@ const tools_in_subfolders = ['vast', 'vcreate', 'vdoc', 'vpm', 'vvet', 'vwhere']
 const non_packaged_tools = ['gen1m', 'gen_vc', 'fast', 'wyhash']
 
 fn main() {
-	util.ensure_modules_for_all_tools_are_installed('-v' in os.args)
+	if os.getenv('VTEST_SANDBOXED_PACKAGING') == '' {
+		util.ensure_modules_for_all_tools_are_installed('-v' in os.args)
+	}
 	args_string := os.args[1..].join(' ')
 	vexe := os.getenv('VEXE')
 	vroot := os.dir(vexe)
-- 
2.39.3 (Apple Git-146)
